src/app/public/modules/file-attachment/file-drop-control.component.html

```diff
@@ -0,0 +1,24 @@
+<sky-file-drop
+  [acceptedTypes]="acceptedTypes"
+  [allowLinks]="allowLinks"
+  [maxFileSize]="maxFileSize"
+  [minFileSize]="minFileSize"
+  [multiple]="multiple"
+  [noClick]="noClick"
+  [validateFn]="validateFn"
+  (controlsTouched)="handleControlTouch()"
+  (filesChanged)="filesUpdated($event)"
+  (linkChanged)="linkUpdated($event)"
+>
+  <ng-content></ng-content>
+</sky-file-drop>
+
+<ng-content select="sky-alert"></ng-content>
+
+<div *ngFor="let item of allItems">
+  <sky-file-item
```

Comment:
Now that we're moving the `sky-file-item`s to this template, how would a consumer load the new control with values already added? I tried to add a `SkyFileDropValue` to the `new FormControl()` statement, but it didn't add any `sky-file-item`s to the template. Do we support programmatically adding files/links?

Reply:
I'll look at this.